src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInNameCheck.java

```diff
@@ -154,6 +158,29 @@
  *   String firstXML; // violation, XML abbreviation is not allowed
  * }
  * </pre>
+ * <p>
+ * To configure to check variables, enforcing no abbreviations
+ * except for variables that are both static and final.
+ * </p>
+ * <p>Configuration:</p>
+ * <pre>
+ * &lt;module name="AbbreviationAsWordInName"&gt;
+ *     &lt;property name="tokens" value="VARIABLE_DEF"/&gt;
+ *     &lt;property name="ignoreFinal" value="false"/&gt;
+ *     &lt;property name="ignoreStatic" value="false"/&gt;
+ *     &lt;property name="ignoreStaticFinal" value="true"/&gt;
+ *     &lt;property name="allowedAbbreviationLength" value="0"/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class MyClass {
+ *     public int AAAA = 1;              // violation
+ *     public final int BBBB = 2;        // violation
+ *     public static int CCCC = 3;       // violation
+ *     public static final int DDDD = 4; // OK, ignored
```

Comment:
please update some previous example to show user that `public static final .....` is ignored if any other property `ignore...` are true.
Please use look at previous examples they try to use real-life usecases in names, please do the same.
User need to see the same code line to be with and without violation to see how property works.

Reply:
@strkkk , @pbludov , please be attentive for this point in documentation.
We should not return to documentation update in future. It is better to do all in first iteration, while we are all in context of issue.

Reply:
> please update some previous example to show user that `public static final .....` is ignored if any other property `ignore...` are true.

I added both `final` and `static final` declarations to the existing examples to highlight that they are ignored.

> Please use look at previous examples they try to use real-life usecases in names, please do the same.

Fixed.

> User need to see the same code line to be with and without violation to see how property works.

I added a copy of this example with a different configuration to show the difference in violations. Let me know if it needs to be more detailed than that.

Reply:
> We should not return to documentation update in future. It is better to do all in first iteration, while we are all in context of issue.

Also since we are here about documentation, am I allowed to take over #7566 as well? Then we can finish up all the documentation problems of this check until next time.

Reply:
yes, you are welcome to fix doc for this Check, but please in separate PR.
thanks a lot for help.